File: mobile/lib/enums/tipo_usuario_constants.dart
// lib/enums/tipo_granja_constants.dart

/// Enum representando o id referente aos tipos de usuario.
enum TipoUsuarioId {
  dono,
  funcionario
}

/// Mapeamento de TipoUsuarioId para valores inteiros.
const Map<TipoUsuarioId, int> tipoUsuarioIdToInt = {
  TipoUsuarioId.dono: 1,
  TipoUsuarioId.funcionario: 2
};

File: web/src/app/constants/tipoUsuarioConstants.ts
export enum TipoUsuarioId {
  DONO = 1,
  FUNCIONARIO = 2,
}

export const TipoUsuarioIdDescriptions: Record<TipoUsuarioId, string> = {
  [TipoUsuarioId.DONO]: 'Administrador',
  [TipoUsuarioId.FUNCIONARIO]: 'Funcionário',
};
